src/supertux/gameconfig.cpp

```diff
@@ -68,8 +68,10 @@ Config::Config() :
   locale(),
   keyboard_config(),
   joystick_config(),
-  mobile_controls(SDL_GetNumTouchDevices() > 0),
+#ifdef __ANDROID__
+  mobile_controls(true),
   m_mobile_controls_scale(1),
+#endif
```

Comment:
This was already adjusted when the Android builds were first setup. This will not work on non-Android touch devices. It's better to let SDL detect if there is touch support on the current OS.

Reply:
@MatusGuy Only initially the configuration sets the option to the number of touch devices, you can always disable it from options.

Reply:
oh yeah right sorry

Reply:
@Semphriss i changed it